--- build1.sh	2023-10-05 10:42:34.571168873 -0500
+++ build.sh	2023-10-05 10:43:02.881855873 -0500
@@ -30,7 +30,7 @@
 
 # restore packages
 echo "dotnet msbuild build/build.proj /t:Restore /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta"
-dotnet msbuild build/build.proj /t:Restore /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta
+dotnet msbuild build/build.proj /t:Restore /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta /v:normal
 
 if [ $? -ne 0 ]; then
 	echo "Restore failed!!"
@@ -39,7 +39,7 @@
 
 # run tests
 echo "dotnet msbuild build/build.proj /t:CoreUnitTests /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta"
-dotnet msbuild build/build.proj /t:CoreUnitTests /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta
+dotnet msbuild build/build.proj /t:CoreUnitTests /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta /v:normal
 
 if [ $? -ne 0 ]; then
 	echo "Tests failed!!"
