--- SourceBuildold.props	2023-10-05 09:50:39.602009765 -0500
+++ SourceBuild.props	2023-10-05 09:50:57.601472276 -0500
@@ -21,7 +21,7 @@
     <_targetRidPlatformIndex>$(TargetRid.LastIndexOf('-'))</_targetRidPlatformIndex>
     <TargetArch>$(TargetRid.Substring($(_targetRidPlatformIndex)).TrimStart('-'))</TargetArch>
 
-    <LogVerbosity Condition="'$(LogVerbosity)' == ''">minimal</LogVerbosity>
+    <LogVerbosity Condition="'$(LogVerbosity)' == ''">normal</LogVerbosity>
   </PropertyGroup>
 
   <Target Name="GetRuntimeSourceBuildCommandConfiguration"
